app/models/product_import/spreadsheet_data.rb

```diff
@@ -33,10 +34,14 @@ def shipping_index
 
     private
 
+    def import_into_inventory?
```

Comment:
We have some code duplication here. I'm wondering if the `entry` could know if it holds a producer or distributor.

Reply:
I've removed this method, as we already have it in the settings object. I'll call it from there instead, as it's now done that way in most places where that check is made.